docs/src/examples/components/Attachment/Variations/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import ComponentExample from 'docs/src/components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/src/components/ComponentDoc/ExampleSection'
+
+const Variations = () => (
+  <ExampleSection title="Variations">
+    <ComponentExample
+      title="Actionable"
+      description="An Attachment can be styled to indicate possible user interaction."
```

Comment:
not sure, but maybe we could better advertise to the user that this (and couple of other ones) example is clickable - frankly, was able to infer this fact only by looking on the code :)